subsys/bluetooth/host/hci_core.c

```diff
@@ -6692,3 +6713,20 @@ int bt_le_oob_get_sc_data(struct bt_conn *conn,
 	return bt_smp_le_oob_get_sc_data(conn, oobd_local, oobd_remote);
 }
 #endif
+
+#if defined(CONFIG_BT_PRIVACY)
+int bt_set_rpa_timeout(u16_t timeout)
```

Comment:
Related to some other work (which is incomplete at the  moment) I think we should restrict the timeout to always be larger than CONFIG_BT_CONN_CREATE_TIMEOUT.

For refrence incomplete work is here: https://github.com/zephyrproject-rtos/zephyr/pull/21714

Reply:
OK